cluster/gce/config-default.sh

```diff
@@ -297,3 +297,6 @@ PROVIDER_VARS="${PROVIDER_VARS:-} FLUENTD_GCP_MEMORY_LIMIT FLUENTD_GCP_CPU_REQUE
 PROMETHEUS_TO_SD_ENDPOINT="${PROMETHEUS_TO_SD_ENDPOINT:-https://monitoring.googleapis.com/}"
 PROMETHEUS_TO_SD_PREFIX="${PROMETHEUS_TO_SD_PREFIX:-custom.googleapis.com}"
 ENABLE_PROMETHEUS_TO_SD="${ENABLE_PROMETHEUS_TO_SD:-false}"
+
+# Optional: Run kube-proxy as a DaemonSet if set to true, run as static pods otherwise.
+KUBE_PROXY_DAEMONSET="${KUBE_PROXY_DAEMONSET:-false}" # true, false
```

Comment:
Are we going to enable this in alpha clusters in GKE? Rollback paths in GKE require a new release no matter what. Configuration doesn't help us because we don't release the same versions of kubernetes with different configurations. If we add this config option, I want to remove it as soon as possible in 1.9 and make this the only way to deploy kube-proxy.